app/src/main/res/layout/fragment_list_bill.xml

```diff
@@ -123,50 +126,28 @@
 
-                        <TextView
-                            android:layout_width="match_parent"
-                            android:layout_height="wrap_content"
-                            android:text="@string/title_customer"
-                            android:textColor="@color/colorPrimary"
-                            android:textSize="@dimen/sp_15"/>
-
                         <LinearLayout
                             android:layout_width="match_parent"
                             android:layout_height="wrap_content"
                             android:layout_marginTop="@dimen/dp_5"
                             android:orientation="horizontal">
 
-                            <ImageView
-                                android:layout_width="wrap_content"
-                                android:layout_height="wrap_content"
-                                android:layout_gravity="center_vertical"
-                                app:srcCompat="@drawable/ic_short_hair_24dp"
-                                tools:ignore="ContentDescription"/>
-
-                            <android.support.design.widget.TextInputLayout
+                            <TextView
                                 android:layout_width="0dp"
-                                android:layout_height="wrap_content"
+                                android:layout_height="match_parent"
                                 android:layout_weight="1"
-                                android:paddingEnd="@dimen/dp_8"
-                                android:paddingStart="@dimen/dp_8"
-                                bind:errorText="@{viewModel.customerNameError}">
-
-                                <EditText
-                                    android:layout_width="302dp"
-                                    android:layout_height="37dp"
-                                    android:hint="@string/title_customer_name"
-                                    android:inputType="number"
-                                    android:text="@={viewModel.customer.name}"
-                                    android:textSize="@dimen/sp_12"/>
-                            </android.support.design.widget.TextInputLayout>
+                                android:gravity="center_vertical"
+                                android:text="@string/title_customer"
+                                android:textColor="@color/colorPrimary"
+                                android:textSize="@dimen/sp_15"/>
 
                             <ImageView
                                 android:layout_width="wrap_content"
                                 android:layout_height="wrap_content"
-                                android:layout_gravity="center_vertical"
-                                android:layout_marginEnd="@dimen/dp_10"
-                                android:layout_marginRight="@dimen/dp_10"
+                                android:layout_gravity="right"
```

Comment:
WARNING: [RtlHardcoded] Use "`end`" instead of "`right`" to ensure correct behavior in right-to-left locales
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)